src/store/reducers/reducers.js

```diff
@@ -17,6 +17,18 @@ export function APP(state = defaultState.APP, action) {
   }
 }
 
+function LOCLIB(state = defaultState.LOCLIB, action) {
+  switch (action.type) {
+    case types.INIT_LOC_LIB:
+    {
+      console.log(action.config);
```

Comment:
:ambulance:

Reply:
Cool will create issues to handle address the tests for this reducers